blaze/expr/strings.py

```diff
@@ -87,41 +90,32 @@ class UnaryStringFunction(ElemWise):
     _arguments = '_child',
 
 
-class strlen(UnaryStringFunction):
-
-    def __init__(self, *args, **kwargs):
-        warnings.warn("`strlen()` has been deprecated in 0.10 and will be "
-                      "removed in 0.11.  Use ``str_len()`` instead.",
-                      DeprecationWarning)
-        super(self, strlen).__init__(*args, **kwargs)
-
-
-class str_len(UnaryStringFunction):
+class len(UnaryStringFunction):
     schema = datashape.int64
 
 
-class str_upper(UnaryStringFunction):
+class upper(UnaryStringFunction):
 
     @property
     def schema(self):
         return self._child.schema
 
 
-class str_lower(UnaryStringFunction):
+class lower(UnaryStringFunction):
 
     @property
     def schema(self):
         return self._child.schema
 
-class str_isalnum(UnaryStringFunction): schema = bool_
-class str_isalpha(UnaryStringFunction): schema = bool_
-class str_isdecimal(UnaryStringFunction): schema = bool_
-class str_isdigit(UnaryStringFunction): schema = bool_
-class str_islower(UnaryStringFunction): schema = bool_
-class str_isnumeric(UnaryStringFunction): schema = bool_
-class str_isspace(UnaryStringFunction): schema = bool_
-class str_istitle(UnaryStringFunction): schema = bool_
-class str_isupper(UnaryStringFunction): schema = bool_
+class isalnum(UnaryStringFunction): schema = bool_
```

Comment:
Are these `schema` correct?  Shouldn't they be `@property` like the `class lower` and `class upper`, since `isalpha` on an `Option(string)` column should return an `Option(bool_)` dshape, correct?